### PR TITLE
Correct RichUtils.handleKeyCommand return type in docs.

### PR DESCRIPTION
**Pull request by aviraldg**

### CODE REVIEW

File: docs/APIReference-RichUtils.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ getCurrentBlockType(
 handleKeyCommand(
   editorState: EditorState,
   command: string
-): boolean
+): EditorState
```
Comment (hellendag) [inline docs/APIReference-RichUtils.md]: Should be `?EditorState`, since it may also return null.

**Comment by facebook-github-bot-8** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by hellendag** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-6** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/501366250071747/int_phab) to review.

**Pull request closed without merging**